python/mxnet/optimizer.py

```diff
@@ -645,6 +645,195 @@ def update(self, index, weight, grad, state):
         ftml_update(weight, grad, prev_d, prev_v, prev_z, out=weight,
                     lr=lr, wd=wd, **kwargs)
 
+@register
+class LBSGD(Optimizer):
+    """The Large Batch SGD optimizer with momentum and weight decay.
+
+    The optimizer updates the weight by::
+
+        state = momentum * state + lr * rescale_grad * clip(grad, clip_gradient) + wd * weight
+        weight = weight - state
+
+    For details of the update algorithm see :class:`~mxnet.ndarray.lbsgd_update` and
```

Comment:
@ashokei @zhreshold 
Where is lbsgd_update defined? I don't see it.

Please add proper reference to relevant papar

Reply:
it is the update method in LBSGD class.  we can fix that to resolve to right method, the '_' is misleading.
the paper is here: 
https://arxiv.org/pdf/1708.03888.pdf